portfolio/src/components/Connect.vue

```diff
@@ -219,6 +359,11 @@ export default {
         background-color: #555555;
     }
 
+    /* Have to override this to get the right background color */
+    .theme--dark.v-skeleton-loader .v-skeleton-loader__actions, .theme--dark.v-skeleton-loader .v-skeleton-loader__article, .theme--dark.v-skeleton-loader .v-skeleton-loader__card-heading, .theme--dark.v-skeleton-loader .v-skeleton-loader__card-text, .theme--dark.v-skeleton-loader .v-skeleton-loader__date-picker, .theme--dark.v-skeleton-loader .v-skeleton-loader__list-item, .theme--dark.v-skeleton-loader .v-skeleton-loader__list-item-avatar, .theme--dark.v-skeleton-loader .v-skeleton-loader__list-item-avatar-three-line, .theme--dark.v-skeleton-loader .v-skeleton-loader__list-item-avatar-two-line, .theme--dark.v-skeleton-loader .v-skeleton-loader__list-item-text, .theme--dark.v-skeleton-loader .v-skeleton-loader__list-item-three-line, .theme--dark.v-skeleton-loader .v-skeleton-loader__list-item-two-line, .theme--dark.v-skeleton-loader .v-skeleton-loader__table-heading, .theme--dark.v-skeleton-loader .v-skeleton-loader__table-tbody, .theme--dark.v-skeleton-loader .v-skeleton-loader__table-tfoot, .theme--dark.v-skeleton-loader .v-skeleton-loader__table-thead{
```

Comment:
so css has some weight rule, i  wonder if it could be overridden by adding an id or something, since this is too long

Reply:
I tried giving it the background directly but it would only style the container of the component. 

The actual background that I needed to change is pretty deeply nested in the component, so I can't really access it directly to add an ID, but you're right I'll play around with some of the classes to see how small I can get it to.